package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "easy-emebddings",
+  "name": "easy-embeddings",
```

Comment:
Hahaha, I laughed hard at this one ;) Probably makes it really easy to find the package... damn, I shouldn't code at 4am in the morning I guess